sdk1/src/test/java/com/amazonaws/services/dynamodbv2/datamodeling/encryption/DynamoDBEncryptorTest.java

```diff
@@ -198,15 +199,15 @@ public void ensureEncryptedAttributesUnmodified() throws GeneralSecurityExceptio
     Map<String, AttributeValue> encryptedAttributes =
         encryptor.encryptAllFieldsExcept(
             Collections.unmodifiableMap(attribs), context, "hashKey", "rangeKey", "version");
-    String encryptedString = encryptedAttributes.toString();
+    String encryptedString = new TreeMap<>(encryptedAttributes).toString();
```

Comment:
Can we add a comment here about why we are utilizing TreeMap, similar to the comment you have in AttributeValueMarshallerTest.java?

Reply:
Sure, I'll do that!